src/main/java/com/revature/repositories/BatchRepository.java

```diff
@@ -0,0 +1,25 @@
+package com.revature.repositories;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import com.revature.domain.Batch;
+
+@Repository
+public interface BatchRepository extends JpaRepository<Batch, Integer> {
+	
+	Batch findBatchByName(String name);
+	Batch findBatchById(int id);
+	void delete(Batch batchName);
+	Batch save(Batch batchName);
+	List<Batch> findAll();
+	
+//	@Modifying
+//	@Query("update ie_batch Batch set Batch.b_name = ?1, where Batch.b_id = ?2")
+//	void setBatchInfoById(String name, int id);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)